### PR TITLE
Update documentation for installing via Go modules

### PR DESCRIPTION
**Pull request by speijnik**
This updates the installing section of our README, reflecting the switch to Go modules.

### CODE REVIEW

File: README.md
Comment (ghost) [inline README.md:16]: Hi,
when you add the structmapper to your go.mod file directly, this wouldn't solve the dependencies and the go.sum file gets outdated. To fix it, a `go mod tidy` is necessary. Or we go the standard way by using a `go get github.com/anexia-it/go-structmapper` for the latest version.

**Review by ghost — CHANGES_REQUESTED** [review]

File: README.md
Comment (speijnik) [reply README.md:16]: I'd argue that for a Go library's README one doesn't have to explain how Go dependency management works.
Also, this is a slightly adjusted variant of the Installing part of the go-anxcloud README, so we get some consistency into these files.

Long story short: I believe we can keep it this way. If this truly is an issue in your eyes we should bring this up so this gets standardized and applied in all other existing repositories as well.

Comment (LittleFox94) [reply README.md:16]: Randomly jumping in here: I'm not happy with that line in go-anxcloud README either, just didn't touch it yet ^^'

I'd also say it's not really needed to be in the README

Comment (ghost) [reply README.md:16]: Yes, I see it as a problem because this not the official way how to manage go modules. The [go documentation](https://go.dev/doc/modules/managing-dependencies#adding_dependency) says the official way is to use `go get`  or just an `import` statement for the dependency.

TLDR: It's on you to decide how you will go in this PR. I'd think it's better here doing the same as in the go-anxcloud README without mention explicit the `go.mod` file. Yes, we should care about standardization in our repos.

Comment (LittleFox94) [reply README.md:16]: I'll probably remove it from go-anxcloud at some point though

Comment (speijnik) [reply README.md:16]: The idea of these PRs is to unblock us by getting everything updated. The documentation PR here blocks the release right now.
Don't get me wrong, let's have a conversation about standardization of these files, we should do that and normalize this everywhere. But in this PR I felt the urge to update this as the old documentation wasn't correct anymore, while trying to use the same wording we are using elsewhere and invest as little time as possible to get things moving forward quickly.

Comment (LittleFox94) [reply README.md:16]: I just wanted to make sure it's not in any way because go-anxcloud has it that way, everything else is not my part and I'm fine with whatever way you choose. Sorry if that sounded different

Comment (speijnik) [reply README.md:16]: No, this is entirely fine and we should have this conversation and normalize these repositories. I'm just saying that this PR is not the right place in my eyes, as this blocks a Go modules enabled release.
Anyways, if it's fine with you @anx-dclappier please resolve this conversation and I will go ahead with the merge and we'll resolve the topic of the normalized README in another PR.

Comment (ghost) [reply README.md:16]: Ok, I'm fine with it.

**Review by ghost — APPROVED** [review]

**Pull request merged**
